app/src/main/java/com/a0xffffffff/dinesum/FirebaseManager.java

```diff
@@ -177,11 +186,12 @@ public void onCancelled(DatabaseError databaseError) { //update UI here if error
             public void onDataChange(DataSnapshot dataSnapshot) { //something changed!
                 ArrayList<Request> userRequests = new ArrayList<Request>();
                 for (DataSnapshot requestSnapshot : dataSnapshot.getChildren()) {
-                    Request newRequest = FirebaseManager.parseJson(requestSnapshot);
+                    Request newRequest = parseJson(requestSnapshot);
                     Log.d("Init user requests", "requestID: " + newRequest.getRequestID());
                     userRequests.add(newRequest);
                 }
                 // save nearby requests
+                Collections.reverse(userRequests);
                 RequestTracker.getInstance().setUserRequests(userRequests);
             }
```

Comment:
Do we have to call onRequesterRequestsReady() here?

Reply:
Haven't done other fragments yet, other than RequestFeedFragment. We will need the call once we have other fragments ready